sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -312,6 +312,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val META_DATA_NUM_RETRIES = buildConf("spark.sql.metadata.num.retries")
```

Comment:
Hi, @ulysses-you .
Since you filed [SPARK-28597](https://issues.apache.org/jira/browse/SPARK-28597) as an `Improvement` JIRA and seems to propose this new configuration, could you update the PR title to focus on your new feature?

Reply:
OK, I changed it.